src/MicrosoftAuthFlow.js

```diff
@@ -56,12 +54,22 @@ class MicrosoftAuthFlow {
       }
     }
 
-    if (this.options.authTitle) { // Login with login.live.com
-      const scopes = (this.options.skipTitleAuth) ? ['XboxLive.signin', 'offline_access'] : ['service::user.auth.xboxlive.com::MBI_SSL']
-      this.msa = new LiveTokenManager(this.options.authTitle, scopes, cache({ cacheName: 'live', username }))
-    } else { // Login with microsoftonline.com
-      const scopes = ['XboxLive.signin', 'offline_access']
-      this.msa = new MsaTokenManager(msalConfig, scopes, cache({ cacheName: 'msa', username }))
+    // prevent breaking changes for v1.7.0 and earlier
```

Comment:
too messy, this should be a breaking change

Reply:
Ok wasn't sure whether it should or not, will remove it